CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing
+
+We welcome contributions from the community. Please read the following guidelines carefully to maximize the chances of your PR being merged.
+
+## Coding Style
+
+- The code is linted using a relatively stringent [golang-ci config](./.golangci.yml). To run this linter (and a few others) use run `make check`. To format your files, you can run `make format`.
```

Comment:
we  don't have `.golangci.yml` :D

Reply:
I keep forgetting hitting "resolve" doesn't add the pending comments typed! sorry, I meant to ack that this is now done.